### PR TITLE
Feature/tao 10285 property removal event

### PR DESCRIPTION
**Pull request by luisfn**
The goal with these changes is to handle the ElasticSearch operation to remove the class property on any already indexed document matching the conditions

- Added method for removal on IndexUpdater
- Moved query execution to separated method
- Added exclusive exception for Removal operation

### CODE REVIEW

**Comment by luisfn** [comment]
Still need to add tests

File: src/IndexUpdater.php [hunk 1]
```diff
@@ -118,6 +103,34 @@ public function updateProperties(array $properties): void
...
+            $name
+        );
+
+        try {
+            $this->executeUpdateQuery($index, $type, $script);
+        } catch (Throwable $e) {
+            throw new FailToRemovePropertyException(
+                sprintf(
+                    'by script: %s AND type: %s',
+                    $result,
```
Comment (Skyld) [inline src/IndexUpdater.php:125]: Maybe my brain has stopped working but where does this variable come from?

Comment (luisfn) [reply src/IndexUpdater.php]: Your brain seems to be working fine :) You are correct. Fixed now

File: test/IndexUpdaterTest.php [hunk 2]
```diff
@@ -119,7 +120,62 @@ public function testDoNotUpdateProperties(array $properties): void
...
+                        'script' => [
+                            'source' => $source
+                        ]
+                    ]
+                ]
+            );
+        $this->sut->deleteProperty($property);
+    }
+
+    public function testExceptionWhenRemovingProperty(): void {
```
Comment (Skyld) [inline test/IndexUpdaterTest.php:151]: Bracket into new line

Comment (luisfn) [reply test/IndexUpdaterTest.php]: Thx. Fixed

**Pull request merged**